File: www/cgi-bin/hrima/post/empty_input/post.php
<?php
$msg = "<html>\n\t<body>\n\t\t<h1>Hello from POST PHP CGI that dosn't take input</h1>\n\t</body>\n</html>";

header("HTTP/1.0 200 OK");
header("Content-Type: text/html");
header("Content-Length: " . strlen($msg));

echo $msg;
?>

File: www/cgi-bin/hrima/post/empty_input/post.py


msg = ""
msg2 = "Hello from POST python CGI that doesn't take input"

i = 0
while i < 1:
    msg = msg + "\n" + msg2
    i += 1

print("HTTP/1.0 200 OfffK\r")
print("Content-Type: text/html\r")
print("Content-Length: " + str(len(msg)) + "\r" )
print("\r")
print(msg, end='')